port 5432 in Port usage definitions (#8101)
chef/automate: components/docs-chef-io/content/automate/ha_aws_deployment_prerequisites.md

| TCP/UDP  | 9638        | Habitat gossip (UDP)                                      |
| TCP      | 7432        | HAProxy, which redirects to PostgreSQL Leader             |
| TCP      | 6432        | Re-elect PostgreSQL Leader if PostgreSQL leader is down   |

> | TCP      | 5432        | Allows PostgreSQL nodes talk to each other in the cluster |

Reviewer: ```suggestion
| TCP      | 5432        | Allows PostgreSQL nodes to connect with each other in cluster |
```
